odoo/tools/config.py

```diff
@@ -115,6 +115,7 @@ def __init__(self, fname=None):
         group.add_option("--without-demo", dest="without_demo",
                           help="disable loading demo data for modules to be installed (comma-separated, use \"all\" for all modules). Requires -d and -i. Default is %default",
                           my_default=False)
+        group.add_option("--skip-auto-install", dest="skip_auto_install", action="store_true", my_default=False, help="skip the automatic installation of modules marked as auto_install")
```

Comment:
It is gonna be saved in the configuration file as
```ini
skip_auto_install = False
```
I don't like double negations.

```suggestion
        group.add_option("--skip-auto-install", dest="auto_install", action="store_false", my_default=True, help="skip the automatic installation of modules marked as auto_install")
```

Reply:
Then using a flag means that in the future we'll have to use `nargs='?'` and allow for values True/False/coma-separated-list-of-modules and a dedicated option checker/formatter to correctly handle it.

It is not infeasible, but it'll be complicated, see `--without-demo`.

The (better code-wise) alternative would be to support a coma-separated-list-of-modules right away.